# HG changeset patch
# User Sean Farley <sean.michael.farley@gmail.com>
# Date 1397871654 18000
#      Fri Apr 18 20:40:54 2014 -0500
# Node ID d903f416ef63711865f0b8325cd5bb06132af03e
# Parent  0e344bcb0edfc864c73ef9072be1698768a94c55
make: use frameworks for lapack

diff --git a/admin/Makefile.build b/admin/Makefile.build
--- a/admin/Makefile.build
+++ b/admin/Makefile.build
@@ -353,13 +353,13 @@ endif
 ###########################################################################
 # The following targets are there to avoid make error (target search) when 
 # only the shared libraries are installed (i.e. the corresponding archive 
 # libraries are missing).
 
--lblas:
+-framework:
 
--llapack:
+Accelerate:
 
 -lslatec:
 
 -lX11:
 
diff --git a/admin/Makefile.def b/admin/Makefile.def
--- a/admin/Makefile.def
+++ b/admin/Makefile.def
@@ -536,11 +536,11 @@ endif
 STDCpp_LIB += @STDCXXLIB@
 
 ###########################################################################
 # Kernel libraries
 
-LEGACY_LIBS = -lslatec -llapack -lblas
+LEGACY_LIBS = -lslatec -framework Accelerate
 _GAG_LIBS = -lggui -lgcore -lgio -lgwcs -lgmath -lgsys -lginc
 ifndef GAG_WIN32
   GAG_LIBS = $(_GAG_LIBS)
 else
   GAG_LIBS = -lgkernel
diff --git a/legacy/slatec/Makefile b/legacy/slatec/Makefile
--- a/legacy/slatec/Makefile
+++ b/legacy/slatec/Makefile
@@ -22,11 +22,11 @@ M-all.o N-all.o O-all.o  Q-all.o R-all.o
 Sd-slatec.o Se-slatec.o Sg-slatec.o Si-slatec.o Sln-slatec.o So-slatec.o    \
 Sp-slatec.o Sqr-slatec.o Ss-slatec.o St-slatec.o Suz-slatec.o T-all.o	    \
 U-all.o V-all.o W-all.o X-all.o Y-all.o Z-all.o d1mach.o i1mach.o pimach.o  \
 r1mach.o dummy.o P-all.o P-value.o Dp-1vlu.o 
 
-LIB_DEPENDS = -llapack -lblas
+LIB_DEPENDS = -framework Accelerate
 
 ifeq ($(GAG_COMPILER_KIND),ifc)
   OPTION_FFLAGS = 
 endif
 ifeq ($(GAG_COMPILER_KIND),ifort)
diff --git a/packages/clic/lib/Makefile b/packages/clic/lib/Makefile
--- a/packages/clic/lib/Makefile
+++ b/packages/clic/lib/Makefile
@@ -62,11 +62,11 @@ CLIC_LIB_OBJECTS = blockdata.o check.o c
         sg_line_table.o table_passband.o clic_index.o
 
 # Mathematical library dependency
 ifeq ($(LINEAR),lapack)
   LINEAR_LIB_OBJECTS = singular.o mth_lapack.o
-  LINEAR_LIB_DEPENDS = -llapack -lblas
+  LINEAR_LIB_DEPENDS = -framework Accelerate
 else 
   ifeq ($(LINEAR),nag)
     LINEAR_LIB_OBJECTS = mth_nag.o
     LINEAR_LIB_DEPENDS = -lnag
   else	
